source/publications.rst

```diff
@@ -2,6 +2,11 @@
 
 Publications
 ============
+.. Note:: 
+	Adam:
+	Is this list up to date? Appears to only go up to 2017 for publications and 2018 for News Articles
+	Might include a DOI or URL for each article for convenience
+	Is there any desire to point users to a list of non-SNL/NREL WEC-Sim articles? Might showcase its use outside of the labs
 
```

Comment:
This is not up to date... and we should update it. I just added a link to https://tethys-engineering.pnnl.gov/signature-projects/wec-sim which includes WEC-Sim papers by non NREL/SNL authors